accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+
+* **Microsoft.Win32.Registry.dll**
+* **Microsoft.Win32.Registry.AccessControl.dll**
+* **Microsoft.Win32.SystemEvents.dll**
+* **System.Diagnostics.EventLog.dll**
+* **System.Diagnostics.PerformanceCounter.dll**
+* **System.DirectoryServices.dll**
+* **System.DirectoryServices.AccountManagement.dll**
+* **System.DirectoryServices.Protocols.dll**
+* **System.IO.FileSystem.AccessControl.dll**
+* **System.IO.FileSystem.DriveInfo.dll**
+* **System.IO.Pipes.AccessControl.dll**
+* **System.Management.dll**
+* **System.Net.Http.WinHttpHandler.dll**
+* **System.Runtime.InteropServices.WindowsRuntime.dll**
+* **System.Security.AccessControl.dll**
+* **System.Security.Cryptography.Cng.dll**
+* **System.Security.Cryptography.ProtectedData.dll**
+* **System.Security.Principal.Windows.dll**
+* **System.ServiceProcess.ServiceController.dll**
+* **System.Threading.AccessControl.dll**
+* **System.Threading.Overlapped.dll**
+
+## Specific APIs
+
+### System
+
+> **OPEN** Should we treat those as Windows-only or should we consider them as
+> partially portable?
+
+* **Console**
+    - `Beep(Int32, Int32)`
+    - `get_CapsLock()`
+    - `get_CursorVisible()`
+    - `get_NumberLock()`
+    - `get_Title()`
+    - `MoveBufferArea(Int32, Int32, Int32, Int32, Int32, Int32)`
+    - `MoveBufferArea(Int32, Int32, Int32, Int32, Int32, Int32, Char, ConsoleColor, ConsoleColor)`
+    - `set_BufferHeight(Int32)`
+    - `set_BufferWidth(Int32)`
+    - `set_CursorSize(Int32)`
+    - `set_WindowHeight(Int32)`
+    - `set_WindowLeft(Int32)`
+    - `set_WindowTop(Int32)`
+    - `set_WindowWidth(Int32)`
+    - `SetBufferSize(Int32, Int32)`
+    - `SetWindowPosition(Int32, Int32)`
+    - `SetWindowSize(Int32, Int32)`
+
+### System.Configuration
+
+> **OPEN** Should we mark the entire type?
+
+* **DpapiProtectedConfigurationProvider**
+    - `Decrypt(XmlNode)`
+    - `Encrypt(XmlNode)`
+
+### System.Diagnostics
+
+* **Process**
+    - `GetProcessById(Int32, String)`
+    - `GetProcesses(String)`
+    - `GetProcessesByName(String, String)`
+    - `set_MaxWorkingSet(IntPtr)`
+    - `set_MinWorkingSet(IntPtr)`
+    - `Start(String, String, SecureString, String)`
+    - `Start(String, String, String, SecureString, String)`
+* **ProcessStartInfo**
+    - `get_Domain()`
+    - `get_LoadUserProfile()`
+    - `get_Password()`
+    - `get_PasswordInClearText()`
+    - `set_Domain(String)`
+    - `set_LoadUserProfile(Boolean)`
+    - `set_Password(SecureString)`
+    - `set_PasswordInClearText(String)`
+* **ProcessThread**
+    - `set_PriorityLevel(ThreadPriorityLevel)`
+    - `set_ProcessorAffinity(IntPtr)`
+
+### System.Drawing
+
+> **OPEN** Should we mark the entire type?
+
+* **Graphics**
+    - `CopyFromScreen(Int32, Int32, Int32, Int32, Size)`
+    - `CopyFromScreen(Int32, Int32, Int32, Int32, Size, CopyPixelOperation)`
+    - `CopyFromScreen(Point, Point, Size)`
+    - `CopyFromScreen(Point, Point, Size, CopyPixelOperation)`
```

Comment:
Don't all of the above work on Linux (and on macOS if the "SYSTEM_DRAWING_COMMON_FORCE_X11" environment variable is set)?

Reply:
Removed